src/python/pants/notes/master.rst

```diff
@@ -5,6 +5,97 @@ This document describes ``dev`` releases which occur weekly from master, and whi
 not undergo the vetting associated with ``stable`` releases.
 
 
+1.14.0.dev3 (1/27/2019)
+-----------------------
+
+New Features
+~~~~~~~~~~~~
+
+* make GoTest subclass PartitionedTestRunnerTaskMixin to test transitively (#7145)
+  `PR #7145 <https://github.com/pantsbuild/pants/pull/7145>`_
+
+* Take deploy_jar_rules into account when detecting duplicates (#7113)
+  `PR #7113 <https://github.com/pantsbuild/pants/pull/7113>`_
+
+* Add Zipkin tracing to pants v1 (#7125)
+  `PR #7125 <https://github.com/pantsbuild/pants/pull/7125>`_
+
+* Allow options to be logged to stats server (#7119)
+  `PR #7119 <https://github.com/pantsbuild/pants/pull/7119>`_
+
+API Changes
+~~~~~~~~~~~
+
+* Bump pyzipkin to 0.17.0 (#7161)
+  `PR #7161 <https://github.com/pantsbuild/pants/pull/7161>`_
+
+* remove support for using the pants native toolchain with distutils Extensions in setup.py (#7126)
+  `PR #7126 <https://github.com/pantsbuild/pants/pull/7126>`_
+
+* Bump cucumber version to get jar with correct unicode encoding  (#7134)
+  `Issue #7123. <https://github.com/pantsbuild/pants/issues/7123.>`_
+  `PR #7134 <https://github.com/pantsbuild/pants/pull/7134>`_
+
+Bugfixes
+~~~~~~~~
+
+* blacklist idea plugin gen from faulty earlier edit (#7157)
+  `PR #7157 <https://github.com/pantsbuild/pants/pull/7157>`_
+
+* Add Python 3 support to Pytest with V2 engine (#7153)
+  `PR #7153 <https://github.com/pantsbuild/pants/pull/7153>`_
+  `PR #6098 <https://github.com/pantsbuild/pants/pull/6098>`_
```

Comment:
I don't think you need to reference #6098, as it is merely precedent for why the changes in #7153 work, which is why I linked to it in the PR description.